cfg/supervision-128k.cfg

```diff
@@ -33,5 +33,5 @@ SEGMENTS {
     ZEROPAGE: load = RAM,      type = bss, define   = yes;
     DATA:     load = RAM,      type = bss, define   = yes, offset = $0200;
```

Comment:
In general I don't understand the meaning of this line.
* Why is the `DATA` segment of type `bss` instead of type `rw`?
* The `RAM` memory area starts at $0000. So the `DATA` segment starts at $0200. Is that desired? Why?